pages/create.tsx

```diff
@@ -1,11 +1,13 @@
-import React from "react";
-
+import Link from "next/link";
 import MultisigForm from "../components/forms/MultisigForm";
 import Page from "../components/layout/Page";
 import StackableContainer from "../components/layout/StackableContainer";
 
 const CreatePage = () => (
   <Page>
+    <Link href="/" style={{ color: "white", fontSize: 20, textDecoration: "none" }}>
+      ← Back to Home
+    </Link>
```

Comment:
Here the user loses the current inputs, right? Can we add a confirmation in this case, similar to what we do when switching chains?